src/plugins/fs/ext.c

```diff
@@ -19,6 +19,7 @@
 
 #include <blockdev/utils.h>
 #include <check_deps.h>
+#include <regex.h>
```

Comment:
Please use the [regular expressions implementation from GLib](https://developer.gnome.org/glib/2.54/glib-Perl-compatible-regular-expressions.html)